package.json

```diff
@@ -104,12 +106,14 @@
     "eslint-plugin-storybook": "^0.6.11",
     "eslint-plugin-unused-imports": "^2.0.0",
     "eslint-plugin-vitest": "^0.0.53",
+    "js-dataverse": "file:./js-dataverse-2.0.0.tgz",
```

Comment:
If we do this for years it will add a lot of bloat to the code. I started a Slack thread to discuss this: https://iqss.slack.com/archives/C04MJ76A34Z/p1682594807564159

Reply:
@GPortas thanks for creating this issue about it!

- #77

I'm not worried about this ~10KB file. We'll fix this in the future. Resolving.